vmdb/app/models/miq_request_task.rb

```diff
@@ -62,6 +62,11 @@ def update_request_status
     # Determine status to report
     req_status = status.slice('Error', 'Timeout', 'Warn').keys.first || 'Ok'
 
+    if req_state == "finished" && state != "finished"
+      req_state = (req_status == 'Ok') ? 'provisioned' : "finished"
```

Comment:
@tinaafitz At the base request_task model we cannot default the state to 'provisioned' as not all requests types support that state.  We need to look at other request types like vm migration, reconfigure and automate and figure out a way to derive this value from the sub-classed request task models.  We should also consider standardizing this 'post_execute' state across all request types but I realize that would have a much bigger impact on existing automate models and likely not a candidate for this PR.